app/views/decidim/time_tracker/admin/activities/index.html.erb

```diff
@@ -48,6 +48,10 @@
                   <%= icon_link_to "pencil", edit_task_activity_path(@current_task, activity), t("actions.edit", scope: "decidim.time_tracker"), class: "action-icon--edit" %>
                 <% end %>
 
+                <% if allowed_to? :update, :activity, activity: activity %>
+                  <%= icon_link_to "people", edit_task_activity_form_path(@current_task, activity), t("actions.questionnaire", scope: "decidim.time_tracker"), class: "action-icon--edit" %>
```

Comment:
We need a better icon than "people"

Reply:
What about `document` ?
![image](https://user-images.githubusercontent.com/8806781/97292404-818c4700-184b-11eb-88fc-92dae1ee8941.png)

Reply:
there aso 
![image](https://user-images.githubusercontent.com/1401520/97554571-4028a280-19d7-11eb-8428-abe1e4842439.png)